src/main/scala/com/snowflake/snowpark/internal/OpenTelemetry.scala

```diff
@@ -0,0 +1,41 @@
+package com.snowflake.snowpark.internal
+
+import com.snowflake.snowpark.DataFrame
+import io.opentelemetry.api.GlobalOpenTelemetry
+
+object OpenTelemetry extends Logging {
```

Comment:
One question about how OpenTelemetry trace action function, in Python, our implementation wrap around the action function. I am asking this because we also want to record any exception happened in the span that is not caused by open telemetry, the example to record it is here: https://opentelemetry.io/docs/languages/java/instrumentation/#set-span-status

Reply:
added